lib/App/Mi6.rakumod

```diff
@@ -54,13 +54,25 @@ method !zef-user() {
     if ::("App::Mi6::Fez") ~~ Failure {
         die "To create a distribution for Zef ecosystem, you need to install fez first";
     }
-    if my $user = ::("App::Mi6::Fez").username {
+    if ::("App::Mi6::Fez").username -> $user {
         return $user;
     } else {
         die "To create a distribution for Zef ecosystem, you need to execute 'fez register' and 'fez login' first.";
     }
 }
 
+method !zef-groups() {
+    try require ::("App::Mi6::Fez");
+    if ::("App::Mi6::Fez") ~~ Failure {
+        die "To create a distribution for Zef ecosystem, you need to install fez first";
+    }
+    if ::("App::Mi6::Fez").groups -> @groups {
+        return @groups;
+    } else {
+        die "To create a distribution for Zef ecosystem, you need to execute 'fez register' and 'fez login' first.";
```

Comment:
I'm not belong to any group right now.
Then `::("App::Mi6::Fez").groups -> @groups` is falsy, so I get this `die`.